<?xml version="1.0" encoding="UTF-8"?>
<robot name="cobotta_r">
	
	<link name="world"/>
	<joint name="joint_w" type="fixed">
		<parent link="world"/>
		<child link="base_link"/>
	</joint>
	<link name="base_link">
		<visual>
			<origin rpy="0 0 0" xyz="0 0 0"/>
			<geometry>
				<mesh filename="/base_link.dae" scale="1 1 1"/>
			</geometry>
		</visual>
		<collision>
			<origin rpy="0 0 0" xyz="0 0 0"/>
			<geometry>
				<mesh filename="/base_link.dae" scale="1 1 1"/>
			</geometry>
		</collision>
		<inertial>
			<mass value="1"/>
			<origin rpy="0.000000 0.000000 0.000000" xyz="0.000000 0.000000 0.000000"/>
			<inertia ixx="1" ixy="0" ixz="0" iyy="1" iyz="0" izz="1"/>
		</inertial>
	</link>
	<link name="J1">
		<visual>
			<origin rpy="0 0 0" xyz="0 0 0"/>
			<geometry>
				<mesh filename="/J1.dae" scale="1 1 1"/>
			</geometry>
		</visual>
		<collision>
			<origin rpy="0 0 0" xyz="0 0 0"/>
			<geometry>
				<mesh filename="/J1.dae" scale="1 1 1"/>
			</geometry>
		</collision>
		<inertial>
			<mass value="1"/>
			<origin rpy="0.000000 0.000000 0.000000" xyz="0.000000 0.000000 0.000000"/>
			<inertia ixx="1" ixy="0" ixz="0" iyy="1" iyz="0" izz="1"/>
		</inertial>
	</link>



</robot>
